css/site.styl

```diff
@@ -378,8 +379,18 @@ img
   position: absolute
   top: 0
   left: 0
-  padding: $margin-small
-  z-index: 10
+  padding: $margin-tiny
+  margin-left: $margin-tiny
+  z-index: 100
```

Comment:
oh for the hoverstate but then i moved the hover to the container so i can take this off